### PR TITLE
Tracefoss-286 -  Clean up packages and remove svg-icon

### PR DESCRIPTION
**Pull request by evia-fbedarf**

### CODE REVIEW

**Comment by evia-fbedarf** [comment]
It is best to check the commits because in the first I removed the packages and in the second I replaced the svg-icon with the material icon component.

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +0,0 @@
-# STAGE 1: Build
```
Comment (ds-ext-akononenko) [inline Dockerfile:1]: I'm not sure that dockerfile should be removed. I assum it may be needed for kubernetes.

Comment (evia-fbedarf) [reply Dockerfile:1]: But we don't need it just jet, right?
Or do we need it for the Deployment on Hotel-Budapest?

But if we need it in the future we can just copy it again.

Comment (ds-ext-akononenko) [reply Dockerfile:1]: My understanding is next:
1. Kubernetes run services using docker image
2. Helm chart is used to create kubernetes config
3. ArgoCD uses helm chart to automatically run it
4. In ArgoCD we specify the repo for the application
5. Frontend is not a Java application, and we don't have common way to run the app via gradle

Taking this all into account, with my limited knowledge, I assume that Dockerfile from repo is used by kubernetes in order to run it on pod.

Comment (evia-fbedarf) [reply Dockerfile:1]: Ah, okay makes sense :)

I will re-add it again.

**Review by ds-ext-akononenko — APPROVED** [review]

**Pull request merged**